descriptors/ModuleDescriptor-template.json

```diff
@@ -1,157 +1,167 @@
 {
-  "id": "${artifactId}-${version}",
-  "name": "Calendar module",
-  "requires": [],
+  "id": "@artifactId@-@version@",
+  "name": "Calendar for Service Point Opening Hours",
   "provides": [
     {
       "id": "calendar",
-      "version": "4.0",
+      "version": "5.0",
       "handlers": [
         {
-          "methods": [
-            "GET"
-          ],
-          "pathPattern": "/calendar/periods",
-          "permissionsRequired": [
-            "calendar.opening-hours.collection.get"
-          ]
+          "methods": ["GET"],
+          "pathPattern": "/calendar/calendars",
+          "permissionsRequired": ["calendar.endpoint.calendars.get"]
```

Comment:
What is the intention of including `.endpoint.` in these permission names? What information does it convey?

Reply:
See other permission conversation